client/src/ui/base/_themes.scss

```diff
@@ -314,7 +313,6 @@
 @mixin dark-theme {
   --text-primary: #{$mdn-theme-dark-text-primary};
   --text-secondary: #{$mdn-theme-dark-text-secondary};
-  --text-active: #{$mdn-theme-dark-text-active};
```

Comment:
same, remove $mdn-theme-dark-text-active definition

Reply:
Fixed in 6415c73942.